README.md

```diff
@@ -76,5 +76,7 @@ Reading instructors list:
 
 [CUGraph.info](http://cugraph.info) - all Columbia classes visualized as a graph of prerequisites.
 
+[EquiNote](not yet, but soon.info) - Columbia note exchange for its students.
```

Comment:
Did you finalize the domain? Let me know, I can add. But I'd avoid adding unfinished project.